host/src/commands/prepare.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copy binaries from pkg to os file system, https://github.com/vercel/pkg/issues/342
+ */
+
+import type { Arguments, Argv } from 'yargs';
+import path from 'path';
+import fs from 'fs-extra';
+import { getAd4mHostVersion, getOldestSupportedVersion } from '../utils/config';
+import { ad4mDataDirectory } from '../ad4mDataDirectory';
+import semver from "semver";
+
+type Options = {
+    dataPath?: string;
+};
+
+export const command: string = 'prepare';
+export const desc: string = 'Check the AD4M data directory and remove any old data from previous versions if required';
+
+export const builder = (yargs: Argv) =>
+  yargs
+    .options({
+        dataPath: { 
+            type: 'string', 
+            describe: 'Name of directory to store ad4m data in the systems app data path', 
+            alias: 'rp'
+        },
+    });
+
+function cleanAd4mData(dataPath: string) {
+    const appDataPath = ad4mDataDirectory(dataPath);
+    if (fs.existsSync(appDataPath)) {
+        // Path to our binaries
+        const binaryPath = path.join(appDataPath, 'binary');
+        // Path to our config
+        const configPath = path.join(appDataPath, 'ad4m-host-config.json');
+        // Path to bootstrap seed
+        const bootstrapSeedPath = path.join(appDataPath, 'mainnet_seed.json');
+        // Path to holochain data
+        const holochainDataPath = path.join(appDataPath, 'ad4m', 'h');
+        // Path to languages
+        const languagesPath = path.join(appDataPath, 'ad4m', 'languages');
+        // path to perspectives
+        const perspectivesPath = path.join(appDataPath, 'ad4m', 'perspectives.json');
```

Comment:
Can't we keep the perspectives? I guess the problem is the Neighbourhood status, after deleting all Languages, but maybe we can handle that without deleting?